### PR TITLE
Some more alias -> account renames

### PR DESCRIPTION
**Pull request by Thoralf-M**

### CODE REVIEW

**Review by thibault-martinez — DISMISSED** [review]

File: bindings/core/src/method/client.rs [hunk 1]
```diff
@@ -301,7 +301,7 @@ pub enum ClientMethod {
     //////////////////////////////////////////////////////////////////////
     // Node indexer API
     //////////////////////////////////////////////////////////////////////
-    /// Fetch alias/basic/NFT/foundry output IDs
+    /// Fetch account/basic/NFT/foundry output IDs
```
Comment (DaughterOfMars) [inline bindings/core/src/method/client.rs]: can it do anchor/delegation?

Comment (Thoralf-M) [reply bindings/core/src/method/client.rs]: Added them

Comment (DaughterOfMars) [reply bindings/core/src/method/client.rs]: there are other comments that might be missing them too in these changes

Comment (Thoralf-M) [reply bindings/core/src/method/client.rs]: Updated another place, one is left, but there is a TODO since we didn't filter these types out yet

Comment (DaughterOfMars) [reply bindings/core/src/method/client.rs]: Alrighty

**Review by DaughterOfMars — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**